src/content/en/fundamentals/engage-and-retain/push-notifications/faq.md

```diff
@@ -0,0 +1,154 @@
+Feel free to [raise an issue](https://github.com/gauntface/web-push-book/issues) if you have
+anything you think should be mentioned with this question.
+
+## Why is this Any Better than Web Sockets?
+
+The reason web push is preferable is that a service worker can be brought to
+life, when the browser window is closed. A web socket will only live as long as the browser and
+web page is kept open.
+
+## What is the deal with GCM, FCM, Web Push and Chrome?
+
+This question has a number of facets to it and the easiest way to explain is to
+step through the history of web push and Chrome (Don't worry it's short).
+
+##### December 2014
+When Chrome first implemented web push, Chrome used Google Cloud Messaging (or GCM as it's
+known) to power the sending of push messages from the server to the browser.
+
+This **was not web push**. There are a few reasons this early set-up of Chrome and GCM wasn't
+"real" web push.
+
+1. GCM requires developers to set up an account on the Google Developers Console.
+1. Chrome + GCM needed a special sender ID to be shared by a web app to be able to set up
+messaging correctly.
+1. GCM's servers accepted a custom API request that wasn't a web standard.
+
+##### July 2016
+In July a new feature in web push landed - Application Server Keys (or VAPID as the spec is
+known). When Chrome added support for this new API, anyone who used it will get Chrome to use
+Firebase Cloud Messaging (also known as FCM) instead of GCM. This is important for a few
+reasons.
+
+1. Chrome + Application Sever Keys **do not** need any kind of project to be setup with Google
+or Firebase, it'll just work.
+1. FCM supports the *web push protocol*, which is the API that all web push services will
+support. This means that regardless of what push service a browser uses, you just make the same
+kind of request and it'll send the message.
+
+##### Why is it confusing today?
+There is a large amount of confusion now that content has been written on the topic of web
+push, much of which references GCM or FCM. If content references GCM, you should probably treat
+it as a sign that it's either old content OR it's focusing too much on Chrome (I'm guilty of
+doing this in a number of old posts).
+
+Instead, think of web push as consisting of a browser, which uses a push service to manage
+sending and receiving message, where the push service will accept a "web push protocol"
+request. If you think in these terms, you can ignore which browser and which push service it's
+using and get to work.
+
+This book has been written to focus on the standards approach of web push and purposefully
+ignores anything else.
+
+The *only* time you should care about this back story is if and when you want to support older
+versions of Chrome, Samsung Internet browser and / or Opera for Android, all of whom use the
+older GCM trick / proprietary API's / hacks **but this is strongly discouraged**.
```

Comment:
I combined these two paragraphs because they're part of the same idea. 

It has more symbols that you need to remove.

'But this is...' is a new sentence.